app/code/Magento/Cron/Observer/ProcessCronQueueObserver.php

```diff
@@ -151,7 +169,9 @@ public function __construct(
         \Magento\Framework\Stdlib\DateTime\DateTime $dateTime,
         \Magento\Framework\Process\PhpExecutableFinderFactory $phpExecutableFinderFactory,
         \Psr\Log\LoggerInterface $logger,
-        \Magento\Framework\App\State $state
+        \Magento\Framework\App\State $state,
+        StatFactory $statFactory,
+        \Magento\Framework\Lock\LockManagerInterface $lockManager
```

Comment:
Shouldn't BC be preserved?

Reply:
@adrian-martinez-interactiv4, We treat Observers and Plugins as internal implementation and don't try to preserve BC for them